### PR TITLE
Create functionality list

### PR DESCRIPTION
**Pull request by kalbermattenm**
Demo here: http://sitn.ne.ch/web/gmf/pages/functionalities.html

@rbovard, what do you think about it ? (currently only in English...)

### CODE REVIEW

File: content/pages/3_functionalities.md [hunk 1]
```diff
@@ -1,8 +1,107 @@
...
+
+### ![Functionalities description]({filename}/images/1_2_search.png)  Full text search :
+
+* Data
+* Layers
+* Themes
+
+### ![Functionalities description]({filename}/images/1_3_navigation.png)  Navigation :
+
+* Zoom & Pan
```
Comment (rbovard) [inline content/pages/3_functionalities.md]: `pan`

File: content/pages/3_functionalities.md [hunk 2]
```diff
@@ -1,8 +1,107 @@
...
+* Navigation bar
+* Geolocation
+
+### ![Functionalities description]({filename}/images/1_4_query.png)  Query :
+
+* Point & rectangular interrogation 
+* Table and/or popup results
+* Live Raster & coordinate interrogation
```
Comment (rbovard) [inline content/pages/3_functionalities.md]: `raster`

File: content/pages/3_functionalities.md [hunk 3]
```diff
@@ -1,8 +1,107 @@
...
+
+### ![Functionalities description]({filename}/images/1_5_data.png)  Data :
+
+* Internal  WMS & WMTS
+* WMS-T
+* External data (WMS-KML)
```
Comment (rbovard) [inline content/pages/3_functionalities.md]: WMS, KML

File: content/pages/3_functionalities.md [hunk 4]
```diff
@@ -1,8 +1,107 @@
...
+
+### ![Functionalities description]({filename}/images/1_9_gis.png)  GIS functionalities :
+
+* SQL queries & filters 
+* Time sliders 
+
+### ![Functionalities description]({filename}/images/1_10_misc.png)  Misc :
+
+* Profile
+* StreetView integration
```
Comment (rbovard) [inline content/pages/3_functionalities.md]: `Google Street View`

File: content/pages/3_functionalities.md [hunk 5]
```diff
@@ -1,8 +1,107 @@
...
+* Permalink
+* Permalink shortener
+
+---
+
+## GMF administration :
+
+### ![Functionalities description]({filename}/images/2_1_configuration.png)  User administration :
+
+* Layer configuration
```
Comment (rbovard) [inline content/pages/3_functionalities.md:79]: `Layers`

File: content/pages/3_functionalities.md [hunk 6]
```diff
@@ -1,8 +1,107 @@
...
+* Group configuration
```
Comment (rbovard) [inline content/pages/3_functionalities.md:80]: `Groups`

File: content/pages/3_functionalities.md [hunk 7]
```diff
@@ -1,8 +1,107 @@
...
+* Theme configuration
```
Comment (rbovard) [inline content/pages/3_functionalities.md:81]: `Themes`

File: content/pages/3_functionalities.md [hunk 8]
```diff
@@ -1,8 +1,107 @@
...
+* Data source configuration
```
Comment (rbovard) [inline content/pages/3_functionalities.md:82]: `sources`

File: content/pages/3_functionalities.md [hunk 9]
```diff
@@ -1,8 +1,107 @@
...
+
+### ![Functionalities description]({filename}/images/2_2_authentication.png)  Authentication : 
+
+* LDAP, NTLM & other interaction
```
Comment (rbovard) [inline content/pages/3_functionalities.md]: `interactions`?

File: content/pages/3_functionalities.md [hunk 10]
```diff
@@ -1,8 +1,107 @@
...
+
+## GMF server :
+
+### ![Functionalities description]({filename}/images/3_1_pdf.png)  PDF reporting
+
+### ![Functionalities description]({filename}/images/3_2_webmapping.png)  Webmapping backends :
+
+* MapServer
+* QGIS Server
+* ArcGIS
```
Comment (rbovard) [inline content/pages/3_functionalities.md]: `ArcGIS for Server`

File: content/pages/3_functionalities.md [hunk 11]
```diff
@@ -1,8 +1,107 @@
...
+### ![Functionalities description]({filename}/images/1_10_misc.png)  Misc :
+
+* Profile
+* StreetView integration
+* Permalink
+* Permalink shortener
+
+---
+
+## GMF administration :
```
Comment (rbovard) [inline content/pages/3_functionalities.md]: `GeoMapFish`

File: content/pages/3_functionalities.md [hunk 12]
```diff
@@ -1,8 +1,107 @@
...
+### ![Functionalities description]({filename}/images/2_3_roles.png)  Roles : 
+
+* Roles configuration
+* User configuration
+* User restriction areas
+* User functionalites
+
+---
+
+## GMF server :
```
Comment (rbovard) [inline content/pages/3_functionalities.md]: `GeoMapFish`

**Review by rbovard — COMMENTED** [review]
Thanks for this work @kalbermattenm and sorry for the delay...

Expect those little typos I marked, I have two comments:
* I'm not a big fan of headers with icons (maybe you can put them under?
* Maybe remove the `:` in the headers?

Feel free to take this in consideration :)

**Comment by kalbermattenm** [comment]
OK, I will remove the `:` and try to make a better proposition concerning these icons...

**Comment by rbovard** [comment]
Maybe you can let the proposition for the icons for another time and do only the little changes?

**Comment by kalbermattenm** [comment]
Yeah... I am just going to leave this PR open as a reminder...

> do only the little changes?

Which changes are you expecting from me ? Should I apply your review and refactor the icon / table stuff later (would be cool, because I do not have any time to do more then applying a slight review...)

**Comment by rbovard** [comment]
> Which changes are you expecting from me ? Should I apply your review and refactor the icon / table stuff later (would be cool, because I do not have any time to do more then applying a slight review...)

Exactly.

> I am just going to leave this PR open as a reminder...

You can create an issue with this topic so you can close this PR.

**Comment by kalbermattenm** [comment]
OK, so I am going to apply your review and merge this PR (tomorrow...)

**Comment by rbovard** [comment]
Cool, thanks!

File: content/pages/3_functionalities.md (hunk 5 above)
Comment (kalbermattenm) [reply content/pages/3_functionalities.md:79]: It is singular here... You do not use plural in that case...

File: content/pages/3_functionalities.md (hunk 6 above)
Comment (kalbermattenm) [reply content/pages/3_functionalities.md:80]: Same here

File: content/pages/3_functionalities.md (hunk 7 above)
Comment (kalbermattenm) [reply content/pages/3_functionalities.md:81]: Same here

File: content/pages/3_functionalities.md (hunk 8 above)
Comment (kalbermattenm) [reply content/pages/3_functionalities.md:82]: Same here

**Comment by kalbermattenm** [comment]
Review applied, @rbovard OK to merge ?

(This only refactors functionalities in English. I will have to do it in French and German, but I rather do that once I am sure of how the structure is going to look like)

**Comment by rbovard** [comment]
Thanks, LGTM! Please merge.

**Pull request merged**